netbox_data_import/templates/netbox_data_import/_import_progress.html

```
@@ -0,0 +1,38 @@
+{# SPDX-License-Identifier: Apache-2.0 #}
+{# Copyright (C) 2025 Marcin Zieba <marcinpsk@gmail.com> #}
+<div id="import-progress"
+     {% if is_active %}
+     hx-get="{% url 'plugins:netbox_data_import:import_progress_status' pk=job.pk %}"
+     hx-trigger="load, every 1s"
+     hx-swap="outerHTML"
+     {% endif %}>
```

HTMLHint (1.9.2)

[error] Duplicate of attribute name [ {% ] was found.

(attr-no-duplication)